lib/views/widgets/rich_text_option_bottom_sheet.dart

```diff
@@ -0,0 +1,256 @@
+                color: CommonColor.colorIconSelect,
+                fontSize: 16,
+                fontWeight: FontWeight.w500,
+              ),
+            ),
+            LimitedBox(
+              maxWidth: 28,
+              maxHeight: 28,
+              child: SvgPicture.asset(
+                _imagePaths.icArrowRight,
+                package: packageName,
+                fit: BoxFit.contain,
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+
+  Widget _buildAlignStyle() {
+    return _buildBorderContainer(
+      Row(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildIconButton(false, () {}, _imagePaths.icAlignLeft),
+          _buildVerticalDivider(),
+          _buildIconButton(false, () {}, _imagePaths.icAlignCenter),
+          _buildVerticalDivider(),
+          _buildIconButton(false, () {}, _imagePaths.icAlignRight),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildColorStyle() {
+    return _buildBorderContainer(
+      Row(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildIconButton(false, () {}, _imagePaths.icTextColor),
+          _buildVerticalDivider(),
+          _buildIconButton(false, () {}, _imagePaths.icBackgroundColor),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildFormatStyle() {
+    return _buildBorderContainer(
+      Row(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildIconButton(false, () {}, _imagePaths.icIndentFormat),
+          _buildVerticalDivider(),
+          _buildIconButton(false, () {}, _imagePaths.icOutDentFormat),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildOrderListStyle() {
+    return _buildBorderContainer(
+      Row(
+        mainAxisSize: MainAxisSize.max,
+        mainAxisAlignment: MainAxisAlignment.center,
+        children: [
+          _buildIconButton(false, () {}, _imagePaths.icBulletOrder),
+          _buildVerticalDivider(),
+          _buildIconButton(false, () {}, _imagePaths.icNumberOrder),
+        ],
+      ),
+    );
+  }
+
+  Widget _buildVerticalDivider() => Container(
+        width: 1,
+        color: CommonColor.colorBorderGray,
+      );
+
+  Widget _buildIconButton(
+    bool isSelected,
+    VoidCallback onTap,
+    String asset,
+  ) {
+    return Expanded(
+      child: InkWell(
+        onTap: onTap,
+        child: Container(
+          color: isSelected ? CommonColor.colorBackgroundSelect : Colors.white,
+          padding: const EdgeInsets.symmetric(vertical: 8, horizontal: 10),
+          height: double.infinity,
+          child: LimitedBox(
+            maxWidth: 28,
+            maxHeight: 28,
+            child: SvgPicture.asset(
+              asset,
+              color: isSelected ? CommonColor.colorIconSelect : null,
+              package: packageName,
```

Comment:
why need use it?

Reply:
Need it for load image form package without defining it on `pubspec.yaml` file of project implement